sickbeard/providers/btn.py

```diff
@@ -73,78 +73,89 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint:disable=too-many
         :returns: A list of search results (structure)
         """
         results = []
-        self._check_auth()
+        if not self._check_auth():
+            return results
 
         # Search Params
-        search_params = {
-        }
+        search_params = {'age': '<=10800'}  # Results from the past 3 hours
 
-        # age in seconds
-        if age:
-            search_params['age'] = '<=' + str(int(age))
+        for mode in search_strings:
+            items = []
+            logger.log('Search mode: {0}'.format(mode), logger.DEBUG)
 
-        if search_strings:
-            search_params.update(search_strings)
-            logger.log('Search string: {0}'.format(search_strings), logger.DEBUG)
+            for search_string in search_strings[mode]:
```

Comment:
If search_string is really not needed, we can do: for _ in search_strings[mode].
But I don't really see a reason to loop over the search_strings? Then might better remove it all together, to prevent future confusion?